.github/workflows/ci.yml

```diff
@@ -35,6 +35,14 @@ jobs:
       - name: cargo fmt
         run: cargo fmt --all --check
 
+      - name: install ripgrep
+        run: |
+          sudo apt update
```

Comment:
Does this need a `-y` or is that just something people copy and paste and isn’t actually needed?

Reply:
That's a good question. We haven't had `-y` in any of our scripts previously and haven't had a single problem. So maybe GitHub actions have `APT::Get::Assume-Yes` pre-configured? I can look a bit into it later.

Reply:
Indeed that was the case. You can see in the [official GitHub Ubuntu image setup script](https://github.com/actions/runner-images/blob/df722a3cf81bbe65c3b964c9b4830c85451542fc/images/ubuntu/scripts/build/configure-apt.sh#L19):
```sh
# Configure apt to always assume Y
echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
```

So `-y` is implicit for GitHub runners and doesn't have to be explicit.